src/types.ts

```diff
@@ -12,13 +12,32 @@ export type Post = {
   linkToSpecialtyPage?: string
 }
 
+export type CommunityEventCategory = 'tournament' | 'social' | 'learning' | 'generic'
+
+export type CommunityEventCollectionData = {
+  title: string
+  url: string
+  category: CommunityEventCategory
+  startDateTime: string
+  ignoreStartTime?: boolean
+  endDateTime?: string
+  ignoreEndTime?: boolean
```

Comment:
```suggestion
  useEndTime: boolean
```

We may opt to make it non-optional as well, if the default is working to force any value.